src/components/EyesOnMe.js

```diff
@@ -1 +1,23 @@
-// Code EyesOnMe Component Here
+import React from "react"
+
+const handleFocus = (event) => {
+  if (event.currentTarget === event.target) {
+    console.log("Good!")
+  }
```

Comment:
No need for the conditional here:
```suggestion
      console.log("Good!")
```